content/content-strategy/define-user-content-needs/create-actionable-artefacts/user-profiles/index.yml

```diff
@@ -0,0 +1,25 @@
+pagetitle: User profiles
+description: Abstract
+weight: 10
```

Comment:
Check this value against other index.yml files for the topic (incl. subtopics) I suspect this value may cause list order issues.

Reply:
Fixed